util/dataframe_ops.py

```diff
@@ -320,21 +323,29 @@ def check_file_with_tolerances(
         ds_tol = pd.read_csv(tolerance_file_name, index_col=0)
         df_tol = ds_tol * factor
 
-        df_ref = parse_probtest_fof(input_file_ref.path)
+        df_ref_rep, df_ref_veri = parse_probtest_fof(input_file_ref.path)
```

Comment:
Would it make sense to rename `df_ref_veri` into `df_ref_obs` , the same for cur?